src/sentry/static/sentry/app/views/releasesV2/detail/releaseActions.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {browserHistory} from 'react-router';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import Button from 'app/components/button';
+import {IconDelete} from 'app/icons';
+import Confirm from 'app/components/confirm';
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+} from 'app/actionCreators/indicator';
+import Tooltip from 'app/components/tooltip';
+
+import {deleteRelease} from './utils';
+
+type Props = {
+  orgId: string;
+  version: string;
+};
+
+const ReleaseActions = ({orgId, version}: Props) => {
+  const handleDelete = async () => {
+    const redirectPath = `/organizations/${orgId}/releases/`;
+    addLoadingMessage(t('Deleting Release...'));
+
+    try {
+      await deleteRelease(orgId, version);
+      addSuccessMessage(t('Release was successfully removed.'));
+      browserHistory.push(redirectPath);
+    } catch (error) {
+      const errorMessage =
+        error.responseJSON?.detail ?? t('Release could not be be removed.');
+      addErrorMessage(errorMessage);
+    }
+  };
+
+  return (
+    <Wrapper>
+      <Confirm
+        onConfirm={handleDelete}
+        message={t(
+          'Deleting this release is permanent and will affect other projects associated with it. Are you sure you wish to continue?'
+        )}
+      >
+        <div>
+          <Tooltip title={t('You can only delete releases if they have no data.')}>
```

Comment:
```suggestion
          <Tooltip title={t('You can only delete releases if they have no issues or health data.')}>
```

Might help to be clearer here.